src/query-to-elm.ts

```diff
@@ -493,7 +493,7 @@ function translateQuery(uri: string, doc: Document, schema: GraphQLSchema, verb:
     if (field.selectionSet) {
       let isMaybe = false
       if (info_type instanceof GraphQLNonNull) {
-	info_type = info_type['ofType']	
```

Comment:
Same issue as above. Did you remove this due to an error or were you under the assumption it was vestigial?